interfaces/acados_template/setup.py

```diff
@@ -64,27 +64,20 @@
        'cython',
     ],
     package_data={'': [
-        'c_templates_tera/main.in.c',
-        'c_templates_tera/main_sim.in.c',
         'c_templates_tera/Makefile.in',
-        'c_templates_tera/model.in.h',
-        'c_templates_tera/acados_solver.in.c',
-        'c_templates_tera/acados_solver.in.h',
         'c_templates_tera/acados_sim_solver.in.c',
         'c_templates_tera/acados_sim_solver.in.h',
+        'c_templates_tera/acados_sim_solver.in.pxd',
+        'c_templates_tera/acados_solver.in.c',
+        'c_templates_tera/acados_solver.in.h',
+        'c_templates_tera/acados_solver.in.pxd',
         'c_templates_tera/acados_solver_sfun.in.c',
-        'c_templates_tera/p_constraint.in.h',
-        'c_templates_tera/h_constraint.in.h',
-        'c_templates_tera/h_e_constraint.in.h',
-        'c_templates_tera/phi_e_constraint.in.h',
-        'c_templates_tera/phi_constraint.in.h',
+        'c_templates_tera/constraints.in.h',
+        'c_templates_tera/cost.in.h',
+        'c_templates_tera/main.in.c',
+        'c_templates_tera/main_sim.in.c',
         'c_templates_tera/make_sfun.in.m',
```

Comment:
The file is now in `interfaces/acados_template/acados_template/c_templates_tera/matlab_templates/make_sfun.in.m`.
Probably all templates should be included here, i.e. also `matlab_templates`.
I think the failing CI was unrelated to this change.
The mex interface does not use
`interfaces/acados_template/setup.py`.

Reply:
> Probably all templates should be included here, i.e. also matlab_templates.

Ah, I've overlooked these files. Just one quick question: what's the reasoning to add the Matlab/Simulink related files in the Python interface? Wouldn't it be quite odd that someone uses the Python interface to generate mex files?

> I think the failing CI was unrelated to this change.

I believe that archive.ubuntu.com had some issues last week, which might explain the failing pipelines.

Anyway, thanks for your feedback. I will make the proposed change later this week.

Reply:
> Just one quick question: what's the reasoning to add the Matlab/Simulink related files in the Python interface? Wouldn't it be quite odd that someone uses the Python interface to generate mex files?

There might be people prototyping in Python that have the need to export an Sfunction block of their solver, since there is some deployment workflow for their embedded target.
But it doesn't seem very convenient I agree :D